addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -13,16 +13,38 @@
 
     @include print-variable('priority-palette-prefix', $o-palette-priority-prefix);
 
-    // Need the colors and name of each color palette
+    // Need the colors, name, menu background color and footer background color of each color palette
     $palette-names: ();
+    $default-presets: ();
+    $bg-attrs: 'menu', 'footer';
+    @each $attr in $bg-attrs {
+        $default-presets: map-merge($default-presets, ($attr: map-get($o-base-color-palette, $attr)))
+    }
+
     @each $palette-name, $palette in $o-color-palettes {
         @if $palette-name != 'user-palette' {
             @each $key, $value in $palette {
                 @include print-variable('o-palette-#{$palette-name}-#{$key}', $value);
             }
             $palette-names: append($palette-names, $palette-name);
+
+            @each $attr-name, $default-preset-idx in $default-presets {
+                $preset-idx: o-safe-get($palette, $attr-name, $default-preset-idx);
+                $preset: nth($o-color-combinations, $preset-idx);
+                $bg: o-safe-get($palette, 'o-cc#{$preset-idx}-bg', map-get($preset, 'bg'));
+                $bg: o-safe-get($palette, $bg, $bg);
+                @include print-variable('o-palette-#{$palette-name}-#{$attr-name}-bg', $bg);
+            }
         }
     }
+
+    // Need the palette colors used as default menu and footer background color
+    @each $attr-name, $default-preset-idx in $default-presets {
+        $preset: nth($o-color-combinations, $default-preset-idx);
+        $default-bg-color: map-get($preset, 'bg');
+        @include print-variable('default-#{$attr-name}-bg-color', $default-bg-color);
```

Comment:
- Is this for custom palette using the logo suggestion?
- Use same convention: above you used -bg, here -bg-color, choose one. Also start with o-, like the rest.